contrib/gitian-build.sh

```diff
@@ -238,6 +256,10 @@ then
     then
         sudo apt-get install lxc
         bin/make-base-vm --suite trusty --arch amd64 --lxc
+    elif [[ -n "$USE_DOCKER" ]]
+    then
+        sudo apt-get install docker-ce
+        bin/make-base-vm --suite trusty --arch amd64 --docker --docker-image-digest b8855dc848e2622653ab557d1ce2f4c34218a9380cceaa51ced85c5f3c8eb201
```

Comment:
Are you sure that this digest will stay the same? They update the image very often. https://hub.docker.com/r/library/ubuntu/tags/

Reply:
@achow101 Is this hash even supposed to be deterministic or does it just serve the purpose of an unique identifier?

Reply:
I'm not sure.. I think it may just serve as a unique identifier.